client/data/files/types.d.ts

```diff
@@ -0,0 +1,62 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { ApiError } from 'wcpay/types/errors';
+import ACTION_TYPES from './action-types';
+
+export interface File {
+	/**
```

Comment:
🏆 These inline type comments are very helpful. Thanks for taking the time to add them!

Reply:
Just following your example in previous PR's